server/lib/src/valueset/address.rs

```diff
@@ -163,6 +163,27 @@ impl ValueSetT for ValueSetAddress {
         Box::new(self.set.iter().map(|a| a.formatted.clone()))
     }
 
+    fn to_scim_value(&self) -> Option<ScimValue> {
+        Some(ScimValue::MultiComplex(
+            self.set
+                .iter()
+                .map(|a| {
+                    let mut complex_attr = ScimComplexAttr::default();
+
+                    complex_attr.insert("formatted".to_string(), a.formatted.clone().into());
+                    complex_attr
+                        .insert("stretAddress".to_string(), a.street_address.clone().into());
+                    complex_attr.insert("locality".to_string(), a.locality.clone().into());
+                    complex_attr.insert("region".to_string(), a.region.clone().into());
+                    complex_attr.insert("postalCode".to_string(), a.postal_code.clone().into());
+                    complex_attr.insert("country".to_string(), a.country.clone().into());
```

Comment:
can we make these key-strings an enum like we do with `Attribute` so we don't have to go back and fix them later? :D

Reply:
stretaddress seems to also be a typo here

Reply:
👨🏻‍🍳 perfect example 😄

Reply:
Yep, will do .